### DISCUSSION TITLE
Github Action `on: create` not trigerring (again)

### DISCUSSION THREAD

**Discussion by garronej** [community/community#24956] [Actions]
<p>Hello,</p>
<p>I have a project under the form of a template that is supposed to trigger a GitHub Actions workflow when the user click on “use this template”</p>
<p><a href="https://github.com/garronej/ts_ci/blob/ccbbcf384d3bc5a4867fdca50fb91762c9bd103d/.github/workflows/template_initialization.yaml" rel="noopener nofollow ugc">This is the Workflow in question. 5</a></p>
<p>It use to work great but now, when a user click on “use this template” the workflow template_initialization.yaml doesn’t trigger.</p>
<p>Thank you for your help and for the otherwise amassing services.</p>
<p>Best regards</p>

**Comment by garronej** [comment]
<p>Up?</p>
<p>I asked this question already here: <a href="https://github.community/t/github-action-on-create-not-trigerring/139385/4" class="inline-onebox">Github Action `on: create` not trigerring - #4 by garronej</a></p>
<p>It solved my issue then but the issue reappeared… :frowning:</p>

**Comment by garronej** [comment]
<p>Seems like it was just a temporary issue &lt;3</p>

**Comment by garronej** [comment, marked as answer]
<p>Well in the end, after hours of tries and error I got to the conclusion that for a workflow to run after a template creation it must have ran successfully on the last commit made on the source repo.<br>
So I can’t use  <code>on create</code> but I have fond a workaround.</p>

**Reply by b3nk3** [reply]
What is your workaround?

**Reply by hammer-abcfitness** [reply]
It would be great to know what you did to make this work.
It would be even better to have github fix this.

**Reply by garronej** [reply]
I did [this](https://github.com/garronej/ts-ci/blob/5996c2f4d9d564f70fe736f7711d4e0d41bc30d2/.github/workflows/ci.yaml#L14). I didn't say it was a **good** workaround.
